stable/metrics-server/values.yaml

```diff
@@ -48,8 +48,12 @@ replicas: 1
 
 podAnnotations: {}
 #  scheduler.alpha.kubernetes.io/critical-pod: ''
+#  priorityClassName: system-node-critical
 
-# priorityClassName: system-node-critical
+serviceLabels: {}
+# Enable the next two labels to show metrics-server when running kubectl cluster-info
+#  kubernetes.io/cluster-service: "true"
```

Comment:
Any downside to enabling these by default?

Reply:
I chose not to, as I don't want to prescribe my solution.

Reply:
Not adding this label by default is probably reasonable as I believe it has no meaning outside of the `kube-system` namespace.  Before I was aware of this pending change, I opened a PR to add the label to the official example deployment: https://github.com/kubernetes-incubator/metrics-server/pull/252